bulbs/homepage_player/templates/homepage_player/bulbs_homepage_player.html

```diff
@@ -0,0 +1,71 @@
+{% load betty humanize ads static content %}
+
+{% if videos.0 %}
+  <bulbs-video-carousel class="video-carousel homepage-player">
+    <div class="homepage-player-nav">
+      {% block homepage_player_nav %}
+        <div class="homepage-player-branding"></div>
+        <div>VIDEO</div>
+      {% endblock homepage_player_nav %}
+    </div>
+
+    <div class="video-carousel-player">
+      <bulbs-video
+        no-endcard
+        src="{{ VIDEOHUB_BASE_URL }}/video/{{ videos.0.videohub_ref.id }}.json"
+      >
+      </bulbs-video>
+    </div>
+
+    <div class="video-carousel-meta">
+      <!--<campaign-display
+        class="campaign-display-preamble"
+        data-track-action=""
+        data-track-category=""
+        placement="detail"
+        preamble-text="Presented by"
+        src=""
+        name-only
+      >
+      </campaign-display>-->
+      <bulbs-video-meta
+        src="{{ VIDEOHUB_BASE_URL }}/video/{{ videos.0.videohub_ref.id }}.json"
+        share-url="http{% if request.is_secure %}s{% endif %}://{{ request.META.HTTP_HOST }}/v/{{ video.id }}"
+        share-email-message="{{share_message}}"
+        share-twitter-handle="{{twitter_handle}}"
+      >
+      </bulbs-video-meta>
+    </div>
+
+    <hr class="video-carousel-divider"/>
+
+    <bulbs-carousel class="video-carousel-carousel">
+      <header class="video-carousel-header">
+        <span class="video-carousel-header-title">
+          Up Next
+        </span>
+
+        <bulbs-carousel-previous class="video-carousel-previous"></bulbs-carousel-previous>
+        <bulbs-carousel-next class="video-carousel-next"></bulbs-carousel-next>
+      </header>
+
+      <bulbs-carousel-slider>
+        {% for video in videos %}
+          <bulbs-carousel-item
+            {# TODO implement /v/2323 redirect route in urls.py #}
+            {# see: https://trello.com/c/3bT7Xuc9/60 #}
+            share-url="http{% if request.is_secure %}s{% endif %}://{{ request.META.HTTP_HOST }}/v/{{ video.id }}"
+            video-url="{{ VIDEOHUB_BASE_URL }}/video/{{ video.videohub_ref.id }}.json"
+            campaign-url="{# TODO: insert per-item campaign url here. It is okay if this attribute is a blank string. #}"
```

Comment:
@spra85 / @mparent61

Do you know where this will come from?

Will we get the campaign url from the `videohub_ref`
```
campaign-url="{{ video.videohub_ref.tunic_campaign_url }}"
```

or from the on-site video record?

```
campaign-url="{% tunic_campaign_url video.tunic_campaign_id }}"
```